docs/reference/project-reference.md

```diff
@@ -1,3 +1,179 @@
 # Project
 
-::: up42.project.Project
+The Project class enables access to the UP42 [analytics functionality](analytics.md). A project stores workflows and their corresponding job runs.
```

Comment:
nitpick: feel free to ignore completely, but it's not just analytics. Users can also have a workflow comprising just of data blocks as well

Reply:
Yeah, we explain that in https://sdk.up42.com/analytics/
I consider getting data via data blocks a part of our analytics platform still